packages/jest-cli/src/reporters/DefaultTestReporter.js

```diff
@@ -158,16 +153,20 @@ class DefaultTestReporter {
     let filesUpdated = 0;
     let matched = 0;
     let updated = 0;
-    let snapshotsUnlinked = aggregatedResults.snapshotsUnlinked;
+    let snapshotFilesDeleted = aggregatedResults.snapshotFilesDeleted;
+    let warnUpdate = false;
     aggregatedResults.testResults.forEach(result => {
       if (result.snapshotsAdded) {
         filesAdded++;
       }
       if (result.snapshotsUpdated) {
         filesUpdated++;
       }
-      if (result.snapshotsUnlinked) {
-        snapshotsUnlinked++;
+      if (result.snapshotFilesDeleted) {
```

Comment:
here probably should be `result.snapshotFileDeleted` (singular instead of plural)